frontend/packages/dev-console/src/components/import/git/GitTypeSelector.tsx

```diff
@@ -26,6 +27,7 @@ const GitTypeSelector: React.FC<GitTypeSelectorProps> = ({ fieldPrefix }) => {
     setFieldValue(`${fieldPrefix}git.type`, gitType, false);
     setFieldTouched(`${fieldPrefix}git.type`, true, false);
   };
+  const typeValue = get(values, `${fieldPrefix}git.type`);
```

Comment:
```suggestion
  const typeValue = _.get(values, `${fieldPrefix}git.type`);
```
To remain consistent with other files

Reply:
Updated the PR. PTAL.